cmd/snap/cmd_list.go

```diff
@@ -67,12 +67,10 @@ func listSnaps(args []string) error {
 		return err
 	}
 
-	if len(snaps) == 0 {
-		return fmt.Errorf(i18n.G("no snaps found"))
+	if len(snaps) > 0 {
+		sort.Sort(snapsByName(snaps))
```

Comment:
just drop the if altogether; sort et all will work fine if `snaps` is empty / nil